Dockerfile

```diff
@@ -2,26 +2,16 @@
 FROM golang:1.16 as builder
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
+COPY . .
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+RUN make build-manager
 
 # Use distroless as minimal base image to package the manager binary
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
-COPY --from=builder /workspace/manager .
+COPY --from=builder /workspace/bin/manager .
```

Comment:
Does this require also changing ENTRYPOINT?

Reply:
Changing ENTRYPOINT: no.
Yes we can, `bin/manager` is the output path which was generated by kubebuilder.
I'd prefer the binary to be compiled in `.` ?

Reply:
oh, right, misread this. Looks good!

Reply:
For some reason I thought the `.` was before the `/workspace/..` argument :joy: